apps/native-component-list/src/screens/Image/ImageSharedRefFromManipulator.tsx

```diff
@@ -0,0 +1,64 @@
+import { Asset } from 'expo-asset';
+import { Image, ImageRef } from 'expo-image';
+import { FlipType, useImageManipulator } from 'expo-image-manipulator';
+import { useCallback, useState } from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import Button from '../../components/Button';
+import HeadingText from '../../components/HeadingText';
+
+const DEFAULT_IMAGE = Asset.fromModule(require('../../../assets/images/example2.jpg'));
+
+export default function ImageSharedRefFromManipulator() {
+  const [imageRef, setImageRef] = useState<ImageRef | null>(null);
```

Comment:
One thing I don't like in calling images as image refs is that "ref" can be misleading that it's the React ref, while it's a state.